### PR TITLE
Update table styling

### PR DESCRIPTION
**Pull request by ericenns**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

Updates Tables to follow style used in Samples tables.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

image

image

image

image

image

image

image

image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/views/groups/group_links/_invited_groups.html.erb [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
+        bg-slate-50
+        dark:bg-slate-700
+        dark:text-slate-400
+      "
+    >
+      <tr>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.group") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.source") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_granted") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_level") %></th>
```
Comment (deepsidhu85) [inline app/views/groups/group_links/_invited_groups.html.erb:28]: For the access level and expiration columns I think we should set the column width. If the window is resized the columns collapse (see below) This comment applies to the projects invited_groups, members, and group members pages. 

image

File: app/views/data_exports/_data_export.html.erb [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  </td>
+  <td class="px-6 py-3">
+    <%= l(data_export.created_at.localtime, format: :full_date) %>
+  </td>
+  <td class="px-6 py-3">
+    <% unless data_export.expires_at.nil? %>
+      <%= l(data_export.expires_at.localtime, format: :full_date) %>
+    <% end %>
+  </td>
+  <td class="px-6 py-3 text-right">
```
Comment (deepsidhu85) [inline app/views/data_exports/_data_export.html.erb:41]: This comment applies to anywhere we have > 1 actions for a record. I think we should add a bit of spacing between the links. Maybe add a vertical separator.

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
This is looking good just a have a few comments below

File: app/views/groups/group_links/_invited_groups.html.erb (hunk 1 above)
Comment (ericenns) [reply app/views/groups/group_links/_invited_groups.html.erb]: Fixed in [affbd72](https://github.com/phac-nml/irida-next/pull/588/commits/affbd72ca030866300087457173c5410c04bd14c)

File: app/views/data_exports/_data_export.html.erb (hunk 2 above)
Comment (ericenns) [reply app/views/data_exports/_data_export.html.erb]: Fixed in fa6da0e3f3132dff1f660a2258c9b901061be91e

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Review by joshsadam — APPROVED** [review]
Looks good to me, thanks for fixing all these

**Pull request merged**